client/signup/config/flows-pure.js

```diff
@@ -397,6 +397,16 @@ export function generateFlows( {
 			showRecaptcha: true,
 			hideProgressIndicator: true,
 		},
+		{
+			name: 'domain-for-gravatar',
+			steps: [ 'domain-only', 'site-or-domain', 'site-picker' ],
```

Comment:
I think the `site-or-domain` and `site-picker` steps should be removed from this flow, since they're always skipped by default. I wasn't able to remove them because I wasn't sure how to create the domain-only site and how to provide the `siteId` and `siteSlug` flow dependencies without them 🤔